### DISCUSSION TITLE
Need onnx version for desktop app

### DISCUSSION THREAD

**Discussion by ILG2021** [speechbrain/speechbrain#2371] [Feature Request]
### 🚀 The feature

I have develop a desktop app based on speechbrain, use pyinstaller for packing, but the pytorch is very large. So i need onnx version of speechbrain

### Solution outline

use onnx inference code and model. faster for speed and lightweight for package.

**Comment by narugo1992** [comment]
seems most of the models in this library are not onnx supported.

so many if condition used in forward method.

so my suggestion is, just give up this library if you need onnx models.
